[2025-07-13T00:20:19.138+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_ml_dag.detect_flight_anomalies scheduled__2025-07-12T00:00:00+00:00 [queued]>
[2025-07-13T00:20:19.169+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_ml_dag.detect_flight_anomalies scheduled__2025-07-12T00:00:00+00:00 [queued]>
[2025-07-13T00:20:19.188+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-13T00:20:19.227+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): detect_flight_anomalies> on 2025-07-12 00:00:00+00:00
[2025-07-13T00:20:19.254+0000] {standard_task_runner.py:60} INFO - Started process 12369 to run task
[2025-07-13T00:20:19.262+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'flight_ml_dag', 'detect_flight_anomalies', 'scheduled__2025-07-12T00:00:00+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/flight_ml_dag.py', '--cfg-path', '/tmp/tmpn2dljr8l']
[2025-07-13T00:20:19.266+0000] {standard_task_runner.py:88} INFO - Job 172: Subtask detect_flight_anomalies
[2025-07-13T00:20:19.553+0000] {task_command.py:423} INFO - Running <TaskInstance: flight_ml_dag.detect_flight_anomalies scheduled__2025-07-12T00:00:00+00:00 [running]> on host f1b3ce86e5d1
[2025-07-13T00:20:19.653+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_scientist' AIRFLOW_CTX_DAG_ID='flight_ml_dag' AIRFLOW_CTX_TASK_ID='detect_flight_anomalies' AIRFLOW_CTX_EXECUTION_DATE='2025-07-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-12T00:00:00+00:00'
[2025-07-13T00:20:19.667+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-13T00:20:19.783+0000] {logging_mixin.py:188} INFO - ‚úÖ Anomaly detection completed
[2025-07-13T00:20:19.784+0000] {logging_mixin.py:188} INFO - üö® Total anomalies found: 169
[2025-07-13T00:20:19.785+0000] {logging_mixin.py:188} INFO - ‚è±Ô∏è Duration anomalies: 0
[2025-07-13T00:20:19.785+0000] {logging_mixin.py:188} INFO - üìÖ Scheduling anomalies: 77
[2025-07-13T00:20:19.786+0000] {logging_mixin.py:188} INFO - üìä Performance anomalies: 92
[2025-07-13T00:20:19.787+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-13T00:20:19.801+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=flight_ml_dag, task_id=detect_flight_anomalies, execution_date=20250712T000000, start_date=20250713T002019, end_date=20250713T002019
[2025-07-13T00:20:19.855+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-13T00:20:19.888+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
